cypress/support/metaTestHelper.js

```diff
@@ -11,7 +11,7 @@ export const assertCookieValue = (cookieName, value) => {
 };
 
 export const assertCookieExpiryDate = (cookieName, timestamp) => {
-  const testBuffer = 20;
+  const testBuffer = 60;
   cy.getCookie(cookieName).then(c => {
     expect(c.expiry).to.be.within(timestamp - testBuffer, timestamp);
```

Comment:
```suggestion
    expect(c.expiry).to.be.within(
      timestamp - testBuffer,
      timestamp + testBuffer,
    );
```

Im not 100% sure, but I think this might be a rounding bug, this should fix it